src/app/pages/catalogManagement/catalog-management/catalog-management.html

```
@@ -0,0 +1,250 @@
+                            <input type="file" hidden accept="image/*" (change)="onImageUpload($event)" />
+                            <svg class="w-5 h-5 text-gray-700" fill="currentColor" viewBox="0 0 20 20">
+                            <path d="M4 13.586V16h2.414l7.293-7.293-2.414-2.414L4 13.586zM17.707 6.293a1 1 0 0 0 0-1.414l-2.586-2.586a1 1 0 0 0-1.414 0l-1.293 1.293 3.999 3.999 1.294-1.292z"/>
+                            </svg>
+                        </label>
+                    </div>
+            </div>
+            <div class="col-start-6 sm:col-span-3">
+                <label for="Subcategory-name" class="block mb-2 text-sm font-medium text-gray-900 ">Category Name</label>
+                <input [(ngModel)]="newCategory.name"
+                [ngModelOptions]="{standalone:true}"
+                type="text" name="Subcategory-name" id="Subcategory-name" class="shadow-xs bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-600 focus:border-blue-600 block w-full p-2.5 " placeholder="Enter The Subcategory Name" required>
```

HTMLHint (1.5.0)

[error] The id value [ Subcategory-name ] must be unique.

(id-unique)